src/main/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJob.java

```diff
@@ -139,6 +136,7 @@ protected Result uploadFiles() {
             filesToProcess.forEach(file -> {
                 if (sftpService.upload(sftpClass, file)) {
                     FileUtils.deleteFile(file);
```

Comment:
We should delete the file if it fails to upload too so that when it runs next time it does not try to override the file

Reply:
Done

Reply:
File should be deleted if we upload and if we do not so needs to be something like

` if (sftpService.upload(sftpClass, file)) {
                    updateDateSent(file, successUpdateCount, failedUpdateCount, metaData);
                    successCount.incrementAndGet();
                } else {
                    metaData.put("FAILED_TO_UPLOAD_FILE_" + failureCount.incrementAndGet(), file.getName());
                    metaData.put("FAILED_TO_UPDATE_FILE_" + failedUpdateCount.incrementAndGet(), file.getName());
                }
                 FileUtils.deleteFile(file); // to avoid file being overwritten`

Reply:
Done